config/vsphere-csi-driver/patches/deployment-images.yaml

```diff
@@ -5,21 +5,3 @@ metadata:
   namespace: vmware-system-csi
 spec:
   replicas: 1
```

Comment:
I gave up trying to make this kustomizable. kustomize build errors out because it is not valid yaml f we remove the "" and the helmRelease fails with the "" because it becomes a string instead of a int. I can live with it...